Assignment2/irenepg22.ttl

```diff
@@ -0,0 +1,7 @@
+Class01: vocab:includes"Sensor029"
```

Comment:
You are missing final "." characters, and several spaces here and there. You should run an RDF validator.